cl/recap/mergers.py

```diff
@@ -409,26 +412,73 @@ def get_order_of_docket(docket_entries):
     return order
 
 
-def make_recap_sequence_number(de):
+def convert_to_court_timezone(
+    datetime_filed: datetime, court_id: str
+) -> datetime:
+    """Convert a docket entry datetime filed to the court timezone it belongs
+    to.
+
+    :param datetime_filed: The docket entry datetime filed
+    :param court_id: The court id to which docket entries belong, used for
+    timezone conversion.
+    :return: A date object in the court timezone.
+    """
+    court_timezone = COURT_TIMEZONES.get(court_id, "US/Eastern")
+    return do_timezone(datetime_filed, court_timezone)
```

Comment:
OK, this is the other example I was thinking of, where a django helper really makes it hard to understand what's actually happening:
 - `do_timezone` does a bunch of things, then calls
 -  `django.utils.timezone.localtime`, which does a bunch of things and then calls
 -  `datetime.astimezone`. 

It'd be much better to just call `datetime.astimezone` instead of all the django helper stuff that just makes it less clear what is actually happening. I love django, but dang.

Reply:
Yeah, I've changed it to `datetime.astimezone`